### DISCUSSION TITLE
Serverless Function throw DEPLOYMENT_NOT_FOUND after deployment

### DISCUSSION THREAD

**Discussion by linhnguyenltnsolution** [vercel/community#2703] [Help] [answered]
### Summary

I need help with my Serverless function 
It worked well yesterday but today it throw DEPLOYMENT_NOT_FOUND 

I didn't update anything on the code. 

Please help

### Steps to Reproduce

Create Serverless Function with NodeJS 
All code put in API folder

**Comment by smaeda-ks** [comment, marked as answer]
@linhnguyenltnsolution sorry for the trouble. We're investigating this https://www.vercel-status.com/incidents/4fmzhqrbx6x1

**Reply by smaeda-ks** [reply]
Should be resolved now, can you confirm? @linhnguyenltnsolution

**Reply by linhnguyenltnsolution** [reply]
@smaeda-ks it resolved, thank you
